src/test/java/seedu/guest/logic/parser/AddCommandParserTest.java

```diff
@@ -91,81 +103,107 @@ public void parse_compulsoryFieldMissing_failure() {
+        assertParseFailure(parser, INVALID_NAME_DESC + PHONE_DESC_BOB + EMAIL_DESC_BOB
+                        + ROOM_DESC_BOB + DATE_RANGE_DESC_BOB
                 + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB,
                 Name.MESSAGE_CONSTRAINTS);
 
         // invalid phone
-        assertParseFailure(parser, NAME_DESC_BOB + INVALID_PHONE_DESC + EMAIL_DESC_BOB + DATE_RANGE_DESC_BOB
+        assertParseFailure(parser, NAME_DESC_BOB + INVALID_PHONE_DESC + EMAIL_DESC_BOB
+                        + ROOM_DESC_BOB + DATE_RANGE_DESC_BOB
                 + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB,
                 Phone.MESSAGE_CONSTRAINTS);
 
         // invalid email
-        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + INVALID_EMAIL_DESC + DATE_RANGE_DESC_BOB
+        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + INVALID_EMAIL_DESC
+                        + ROOM_DESC_BOB + DATE_RANGE_DESC_BOB
                 + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB,
                 Email.MESSAGE_CONSTRAINTS);
 
+        // invalid room
+        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
+                        + INVALID_ROOM_DESC + DATE_RANGE_DESC_BOB
+                        + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB,
+                Room.MESSAGE_CONSTRAINTS);
+
         // invalid date range
-        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + INVALID_DATE_RANGE_DESC
+        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
+                        + ROOM_DESC_BOB + INVALID_DATE_RANGE_DESC
                 + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB,
                 DateRange.MESSAGE_CONSTRAINTS);
 
         // invalid number of guests
-        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + DATE_RANGE_DESC_BOB
+        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
+                        + ROOM_DESC_BOB + DATE_RANGE_DESC_BOB
                 + INVALID_NUMBER_OF_GUESTS_DESC + IS_ROOM_CLEAN_DESC_BOB,
                 NumberOfGuests.MESSAGE_CONSTRAINTS);
 
         // invalid is room clean
-        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + DATE_RANGE_DESC_BOB
+        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
+                        + ROOM_DESC_BOB + DATE_RANGE_DESC_BOB
                 + NUMBER_OF_GUESTS_DESC_BOB + INVALID_IS_ROOM_CLEAN_DESC,
                 IsRoomClean.MESSAGE_CONSTRAINTS);
 
         // two invalid values, only first invalid value reported
-        assertParseFailure(parser, INVALID_NAME_DESC + PHONE_DESC_BOB + EMAIL_DESC_BOB + DATE_RANGE_DESC_BOB
+        assertParseFailure(parser, INVALID_NAME_DESC + PHONE_DESC_BOB + EMAIL_DESC_BOB
+                        + ROOM_DESC_BOB + DATE_RANGE_DESC_BOB
```

Comment:
Should the indentation of the line follow that of the assertParseSuccess above?

Reply:
Rectified!